docs/_documentations/linking-your-projects.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: docs
+title: Linking your projects
+description: Linking your projects
+keywords: project actions, attach, build, disable, enable, validate, refresh, link, linking your projects, VS Code, Eclipse
+duration: 1 minute
+permalink: linking-your-projects.html
+type: document
+---
+
+# Linking your projects
+
+You can link your projects when you are using Codewind. This feature is available in the [VS Code](#linking-your-projects-in-vs-code) and [Eclipse](#linking-your-projects-in-eclipse) IDEs.
```

Comment:
```
One thing I forgot to mention is that linking doesn't work with the codewind-odo-extension-devfile, it does work with original codewind-odo-extension though
```
Would probably add a sentence: `Project linking is support by all Codewind project types and extensions apart from the "ODO Devfile" extension`

Reply:
Added. Have changed it to: Project linking is supported by all Codewind project types and extensions except for the ODO Devfile extension. ` Apart from` and `except for` can be used interchangeably, but I feel that `except for` spells out the exception better.